tools/sof-dump-status.py

```diff
@@ -2,10 +2,31 @@
 
 import subprocess
 import os
+import sys
+
 from common import format_pipeline, export_pipeline
 
+
+# i.MX specific maps
```

Comment:
Are these maps DeviceTree-specific? If yes then the "Device Tree" keyword should be somewhere in this comment. Maybe even `_DT_` in the constant names?

Reply:
They are. I will definitely add "Device Tree" to the comment so as to make it a bit clearer. I can also add `_DT_` to the names of all the constants if you think it will help others understand the code better but wouldn't it be a bit overkill, especially with the added comment?